docs/source/conf.py

```diff
@@ -0,0 +1,55 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
```

Comment:
You are going to need to uncomment those. 

You also need to modify the path in `os.path.abspath` so it points to the root of this repo.

You can just do like in `conf.py` of the CPP_SPM repo

Reply:
I did change these parts but I suspect I made a mistake and ignore the changes while committing. My bad